libmamba/src/api/channel_loader.cpp

```diff
@@ -86,6 +86,63 @@ namespace mamba
         }
     }
 
+    std::string get_system_certificate()
+    {
+        std::string res = "";
+        if (on_linux)
+        {
+            std::array<std::string, 6> cert_locations{
+                "/etc/ssl/certs/ca-certificates.crt",  // Debian/Ubuntu/Gentoo etc.
+                "/etc/pki/tls/certs/ca-bundle.crt",    // Fedora/RHEL 6
+                "/etc/ssl/ca-bundle.pem",              // OpenSUSE
+                "/etc/pki/tls/cacert.pem",             // OpenELEC
+                "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem",  // CentOS/RHEL 7
+                "/etc/ssl/cert.pem",                                  // Alpine Linux
+            };
+
+            for (const auto& loc : cert_locations)
+            {
+                if (fs::exists(loc))
+                {
+                    res = loc;
+                    break;
+                }
+            }
+
+            if (res.empty())
+            {
+                LOG_ERROR << "No CA certificates found on system";
+                throw std::runtime_error("Aborting.");
+            }
+        }
+        return res;
+    }
+
+    void init_powerloader_ssl(const std::string& value)
+    {
+        powerloader::Context& ctx = Context::instance().plcontext;
+        
+        if (value == "<false>")
+        {
+            ctx.disable_ssl = true;
+            ctx.ssl_ca_info = "";
+        }
+        else if (value == "<system>")
+        {
+            ctx.disable_ssl = false;
```

Comment:
this logic is different now, right? Before it was disableing SSL if no system certificates were found.

Reply:
I dont( think so, for me it was throwing if no system certificate was found on linux, and was relying on the OS to find the certificate otherwise.